### DISCUSSION TITLE
Error on total of shown elements in table

### DISCUSSION THREAD

**Discussion by YnievesDotNet** [Power-Components/livewire-powergrid#1391] [BUG]
I have a problem with powergrid.

when i go to last page of the table, and for example, this page have only 3 rows, when i return to first page show onli 3 rows, and not 10, like has selected in the checkbox...

thanks...
<img width="1267" alt="Captura de pantalla 2024-02-05 a la(s) 12 27 36 p m" src="https://github.com/Power-Components/livewire-powergrid/assets/1451485/7e6b7d53-fadb-48e5-91be-551573178ae0">

**Comment by dansysanalyst** [comment]
I cannot repoduce the problem...

![CleanShot 2024-02-06 at 00 48 39](https://github.com/Power-Components/livewire-powergrid/assets/79267265/f30d9cae-d5a8-40bd-b328-c30bad589a14)

**Reply by YnievesDotNet** [reply]
I have the error on two tables, I can send the video because have production info
